programs/anchor-movie-review-program/src/lib.rs

```diff
@@ -103,3 +103,14 @@ pub struct MovieAccountState {
     pub title: String,
     pub description: String,
 }
+
+/*
+    8 bytes for the anchor discriminator 
+    32 bytes for reviewer Pubkey 
+    1 bytes for the rating
+    4 bytes for the title String (still need to add String length in the account initialization)
+    4 bytes for the description String (still need to add String length in the account initialization),
+ */
+impl Space for MovieAccountState {
+    const INIT_SPACE: usize = 8 + 32 + 1 + 4 + 4;
```

Comment:
Can we use `#[derive(InitSpace)]` for this? https://www.anchor-lang.com/docs/space

Reply:
#[derive(InitSpace)] could definitely be used. However, we would have to give a max length for the String in order for it to work.

Reply:
Thanks. Please do it and use `max_len` (we're using max_len elsewhere and it seems reasonable to be explicit about what we consider reasonable sizes).